policy-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/policy/PolicyTestUtil.java

```diff
@@ -119,7 +119,7 @@ static LeafResource mockResource() {
                 .id("TEST_RESOURCE_ID")
                 .type("data type of the resource, e.g. Employee")
                 .serialisedFormat("none")
-                .connectionDetail(new SimpleConnectionDetail().uri("connection-detail-uri"))
+                .connectionDetail(new SimpleConnectionDetail().serviceName("test-data-service"))
```

Comment:
```suggestion
                .connectionDetail(new SimpleConnectionDetail().serviceName("data-service-mock"))
```
Consistent service name.